### ISSUE TITLE
Standard custom control onChange event 'loses' model path from 1.8.1

### ISSUE THREAD

**Issue by tmfrnz** [davidkpiano/react-redux-form#766]
### The Problem
I implemented RRF's standard custom control "MultiSelect" (immutable) that allows users to select one or more checkbox options. While functioning well up until v1.8.0 as of v1.8.1 the onChange event results in an error in https://github.com/davidkpiano/react-redux-form/blob/master/src/utils/get.js#L13 where our model 'path' now becomes undefined, likely due to changes in src/reducers/forms-reducer.js
https://github.com/davidkpiano/react-redux-form/commit/17f06a5e72f275acc5a2c6d8252125c7a6fb32a0#diff-0deb93befb9f37f19b88b3d449b89065R86

Please note that I have little prior experience with RRF and may be using it wrong or unintendedly. Anyway here our custom form control (simplified)
```jsx
import { Control } from 'react-redux-form/immutable';
const MultiSelectControl = (props) => {
  const {  model, options } = props;
  return (
    <Control
      type="multiselect"
      model={model} 
      component={MultiSelect}
      mapProps={{
        values: (cprops) => cprops.viewValue,
        onChange: (cprops) => props.onChange,
      }}
      controlProps={{
        options, // the initial values
      }}
    />
  );
};
```
and its component (simplified)
```jsx
export default class MultiSelect extends React.Component {

  getNextValues = (checked, option) => {
    const newValue = option.set('checked', checked);
    return this.props.values.set(existingValueIndex, newValue)
  }

  render() {
    const { options, values} = this.props;
    const checkboxes = options.map((option) => {
      const value = values.find((v) => option.get('value') === v.get('value'));
      return option.withMutations((o) =>
        o.set('checked', value ? value.get('checked') : false)
        .set('initialChecked', option.get('checked'))
      );
    });

    return (
      {checkboxes.map( (option, i) => (
        <div key={i}>       
          <input
            id={option.get('value')}
            type="checkbox"
            checked={option.get('checked')}
            onChange={(evt) => {
              if (evt && evt !== undefined) evt.stopPropagation();
              this.props.onChange(this.getNextValues(evt.target.checked, option));
            }}
          />
        <label htmlFor={option.get('value')} >{ option.get('label') }</label>
      </div>
      ))}      
    );
}
```
finally here our form (simplified)
```
import { Form, actions as formActions } from 'react-redux-form/immutable';

class OurForm extends React.Component {
  componentWillMount() {
    this.props.populateForm(this.props.model, this.props.options);
  }
  render() {
    return (
        <Form
          model={this.props.model}
          onSubmit={this.props.onSubmit}
        >
          <MultiSelectControl
            model=".multiselect"
            options={this.props.options}
          />
        </Form>
    );
  }
}
const mapDispatchToProps = (dispatch) => ({
  populateForm: (model, options) => {
    dispatch(formActions.load(model, Map({ values: options })));
  },
});
export default connect(null, mapDispatchToProps)(OurForm);
```

### Expected Behavior
default onChange handler to set view value and handle update 
https://github.com/davidkpiano/react-redux-form/blob/master/src/components/control-component.js#L408

### Actual Behavior
onChange handler's handle update results in error described above

### Reproducible Code Example
Sorry I was not able to reproduce this yet on CodePen

**Comment by davidkpiano** [comment]
Can you try this in the latest version, 1.11.1?

**Comment by tmfrnz** [comment]
Thank you @davidkpiano v1.11.1 appears to have fixed the issue (also the most recent 1.12.1 works)!
